app/templates/app/project_detail.html

```diff
@@ -0,0 +1,82 @@
+{% extends "base.html" %}
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+<br/>
+<div class="project-detail">
+  <p><a href="{% url 'projects' %}">{% trans "Projects" %}</a> > {{ project.name }}</p>
+
+  <div class=project-div-row>
+
+    <div class="col-sm-6 project-detail-left-col">
+      <h1>{{ project.name }}</h1>
+      <a href="{{ project.url }}">{{ project.url }}</a>
+      <p>
+          <a href="{% url 'institution_detail' project.institution.id %}">
+              {{ project.institution.name }}
+          </a>
+      </p>
+      <div>
+          <div class=project-date-row>
+            <div class=project-date-left-col>
+              <p>
+                <span class="icon-text"><i class="project-icon bi-calendar3"></i></span>
+                <span class="icon-text">Start Date: {{ project.start_date|date:"Y-m-d" }}<span class="icon-text">
+              </p>
+            </div>
+            <div class=project-date-right-col>
+              <p>
+                <span class="icon-text"><i class="project-icon bi-calendar3"></i></span>
+                <span class="icon-text">End Date: {{ project.end_date|date:"Y-m-d" }}<span class="icon-text">
+              </p>
+            </div>
+          </div>
+      </div>
+    </div>
+
+    <div class="col-sm-6 project-detail-right-col">
+      {% if project.logo %}
+      <img src="{{ project.logo.url }}" alt="{{ project.name }} logo"
```

Comment:
Please mark the alt text for translation.

Reply:
will do